### ISSUE TITLE
Question regarding Spotify and Brave

### ISSUE THREAD

**Issue by BrutalCSEjziponken** [adirh3/Sonar.AutoSwitch#7]
Hi!

First of all, thanks for this app! Been avoiding Sonar until I found this today!

Now to my questions. How exactly would I do with Spotify and Brave? I cant make it work for either of them. Maybe you could printscreen?

Also, if you could make the program start minimized, that would be great! (win 10)

**Comment by adirh3** [comment]
You should use `overwatch` without thr .`exe`. 
Spotify runs in the background so unless you focus the app while hearing music it will not help much.

What I am doing is have Music Punchy profile as default,  and each game with their respective configuration.

**Comment by BrutalCSEjziponken** [comment]
> You should use `overwatch` without thr .`exe`. Spotify runs in the background so unless you focus the app while hearing music it will not help much.
> 
> What I am doing is have Music Punchy profile as default, and each game with their respective configuration.

Well I realized I can just use Spotify EQ in the "Media" tab in Sonar. And then have a "Game" EQ for my games. That would allow overwatch and spotify to run different EQs at the same time right?

**Comment by adirh3** [comment]
Yes and if you want to switch the Game device EQ on per game basis you can use the Auto Switch.
